### PR TITLE
feat: added domain integration

### PR DESCRIPTION
**Pull request by Pauti**
### Required

- `host` (String) Domain name or IP address.
- `space_id` (String) Mondoo Space Identifier.

### Optional

- `http` (Boolean) Enable HTTP port.
- `https` (Boolean) Enable HTTPS port.

### Read-Only

- `mrn` (String) Integration identifier

### CODE REVIEW

**Comment by Pauti** [comment]
I have read the Mondoo CLA Document and I hereby sign the CLA

**Comment by Pauti** [comment]
recheck

File: internal/provider/integration_domain_resource.go [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+	Http  types.Bool   `tfsdk:"http"`  // http port
+}
+
+func (r *integrationDomainResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_integration_domain"
+}
+
+func (r *integrationDomainResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
```
Comment (chris-rock) [inline internal/provider/integration_domain_resource.go:43]: Markdown description is missing in schema

**Review by chris-rock — APPROVED** [review]

**Comment by chris-rock** [comment]
requires https://github.com/mondoohq/terraform-provider-mondoo/pull/69 to be merged first

**Pull request merged**